docs/how-to/join-a-network.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Joining a network
+sidebarDepth: 0
+description: Learn how to join one of the various Filecoin testnets.
+---
+
+# Joining a network
+
+Once a Lotus node has been initialised on the appropriate [Filecoin network option](https://docs.filecoin.io/how-to/networks/), the instructions within this section will 
+
+### Note: Using the Lotus Node from China
+
+If you are trying to use `lotus` from China, set this **environment variable** on your machine:
+
+```sh
+IPFS_GATEWAY="https://proof-parameters.s3.cn-south-1.jdcloud-oss.com/ipfs/"
```

Comment:
@whyrusleeping Is this still the recommended method of getting proof params in China?